src/Sentry/SessionEndStatus.cs

```diff
@@ -13,10 +13,15 @@ public enum SessionEndStatus
     /// <summary>
     /// Session ended with an unhandled exception.
     /// </summary>
+    Unhandled,
```

Comment:
question: breaking change

We ship this in a major release, so this may be a non-concern.

But inserting an `enum` member is a breaking change for users that have code like:
```CSharp
_ = (SessionEndStatus)1;
```

Well ... I guess it's unlikely that a considerable amount of user code does this ... I just wanted to hear your thoughts.